src/encoder/encoderffmpegcore.cpp

```diff
@@ -194,30 +196,31 @@ void EncoderFfmpegCore::encodeBuffer(const CSAMPLE *samples, const int size) {
 //
 // Currently this method is used before init() once to save artist, title and album
 //
-void EncoderFfmpegCore::updateMetaData(char* artist, char* title, char* album) {
+void EncoderFfmpegCore::updateMetaData(const QString& artist, const QString& title, const QString& album) {
     qDebug() << "ffmpegencodercore: UpdateMetadata: !" << artist << " - " << title <<
              " - " << album;
     m_strMetaDataTitle = title;
```

Comment:
Right now it seems that on linux it uses the ffmpeg engine and on windows (and possibly macos) it uses the lame/vorbis one.
So yes, maybe the ffmpeg implementation should be downgraded and used only if lame/vorbis not compiled in.
Suggestions on how to do so?